### PR TITLE
API provides functionality to create articles

### PR DESCRIPTION
**Pull request by miyeaier**
## User Story 
```
As newspaper api
In order to give articles to end user
I would liie to add functionality to create article in my database

```

Screenshot 2022-04-06 at 14 22 10

### CODE REVIEW

File: spec/requests/api/articles/index_spec.rb [hunk 1]
```diff
@@ -9,7 +12,16 @@
 
   it { is_expected.to have_http_status 200 }
 
-  it 'is expected to return a collection of 5 articles' do
-    expect(response_json['articles'].size).to eq 5
+  # it 'is expected to return a collection of 5 articles' do
+  #   expect(response_json['articles'].size).to eq 5
+  # end
```
Comment (maxarvid) [inline spec/requests/api/articles/index_spec.rb]: Why is this commented out?

File: temp_article_resp.json [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+      {"id": 2, "title": "Sports 2", "body": "Sports 2 body"},
+      {"id": 3, "title": "Sports 3", "body": "Sports 3 body"}
+    ],
+    "business": [
+      {"id": 1, "title": "Business 1", "body": "Business 1 body"},
+      {"id": 2, "title": "Business 2", "body": "Business 2 body"},
+      {"id": 3, "title": "Business 3", "body": "Business 3 body"}
+    ]
+  }
+}
```
Comment (maxarvid) [inline temp_article_resp.json]: I suggest we remove this file. Also, note that article `id` need to be unique.

**Review by maxarvid — CHANGES_REQUESTED** [review]
Almost there.

File: spec/requests/api/articles/index_spec.rb (hunk 1 above)
Comment (Supracorona) [reply spec/requests/api/articles/index_spec.rb]: That is left after Tomas's session:
https://youtu.be/LT24ryaqlFQ?t=851

Should we delete?

File: temp_article_resp.json (hunk 2 above)
Comment (Supracorona) [reply temp_article_resp.json]: I left it on purpose.
--- We talked about it on Tuesday at SCRUM. ---
We will delete.

Comment (miyeaier) [reply temp_article_resp.json]: deleted

File: spec/requests/api/articles/index_spec.rb (hunk 1 above)
Comment (maxarvid) [reply spec/requests/api/articles/index_spec.rb]: deleted

**Review by maxarvid — APPROVED** [review]

**Pull request merged**